test/steps/vorpal/2_when.js

```diff
@@ -17,6 +17,34 @@ import { wrapActionCreator, createCommand } from '../../../src/utils/helpers';
 import execFile from '../../../src/execFile';
 import { getFirstQuotedString } from '../utils';
 
+export function theActionIsCalledWithTheOptionsThePublicKeyTheSignatureAndTheMessage() {
+	const { action, options, publicKey, signature, message } = this.test.ctx;
+	const returnValue = action({ options, publicKey, signature, message });
+	this.test.ctx.returnValue = returnValue;
+	return returnValue.catch(e => e);
+}
+
+export function theActionIsCalledWithTheOptionsTheMessageThePublicKeyAndTheSignature() {
```

Comment:
This duplicates the previous function (but with the arguments in the wrong order).